src/routes/usersRoutes.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
Consider removing the 'use strict' statement as it is not necessary in ES6 modules.

Reply:
![image](https://github.com/mate-academy/node_accounting-app-v2/assets/124785462/991db6b9-fa17-4221-b261-561bced5d374)